src/AWS/Lambda/Events/EventBridge.hs

```diff
@@ -0,0 +1,144 @@
+{-# LANGUAGE ApplicativeDo #-}
+{-# LANGUAGE DeriveTraversable #-}
+{-# LANGUAGE NamedFieldPuns #-}
+
+{-|
+Module      : AWS.Lambda.Events.EventBridge
+Description : Data types for consuming EventBridge events.
+License     : BSD3
+Stability   : stable
+
+Data types for Lambda functions which subscribe to AWS EventBridge
+events.
+
+-}
+
+module AWS.Lambda.Events.EventBridge (
+  EventBridgeEvent,
```

Comment:
It also maintains consistency with `AWS.Lambda.Events.S3.S3Event`, and since most modules appeared to be designed for unqualified import, that's what I went with here. I'll resolve this thread but reopen if you want it changed.